packages/did-provider-ebsi/src/types.ts

```diff
@@ -43,5 +42,26 @@ export interface ICreateIdentifierArgs {
   kms?: string
   alias?: string
   type?: EbsiDidSpecInfo
-  options?: IKeyOpts
+  options?: {
+    methodSpecificId?: string // method specific id for import
+    secp256k1?: IKeyOpts | VerificationMethod
+    secp256r1?: IKeyOpts | VerificationMethod
+  }
+}
+
+export interface ICreateIdentifierOpts {}
+
+export enum KeyType {
+  Secp256k1 = 'Secp256k1',
+  Secp256r1 = 'Secp256r1',
+}
+
+export enum EbsiPublicKeyPurpose {
+  Authentication = 'authentication',
```

Comment:
Did you check these are the only ones available for EBSI?

Reply:
Yes and I didn't come across any other verification relationship. They are consistently mentioned in did authorities (Legal Entity), did creation and registration sections, `AddVerificationMethod` (capabilityInvocation) and `AddVerificationRelationships` (authorization, assetionMethod). In addition, I couldn't find any explicit mention of of all available verification relationships/methods for EBSI.